contracts/SafeBoxETH.sol

```diff
@@ -0,0 +1,91 @@
+pragma solidity 0.6.12;
+
+import 'OpenZeppelin/openzeppelin-contracts@3.2.0/contracts/token/ERC20/ERC20.sol';
+import 'OpenZeppelin/openzeppelin-contracts@3.2.0/contracts/token/ERC20/IERC20.sol';
+import 'OpenZeppelin/openzeppelin-contracts@3.2.0/contracts/token/ERC20/SafeERC20.sol';
+import 'OpenZeppelin/openzeppelin-contracts@3.2.0/contracts/cryptography/MerkleProof.sol';
+import 'OpenZeppelin/openzeppelin-contracts@3.2.0/contracts/math/SafeMath.sol';
+import 'OpenZeppelin/openzeppelin-contracts@3.2.0/contracts/utils/ReentrancyGuard.sol';
+import './Governable.sol';
+import '../interfaces/ICErc20.sol';
+import '../interfaces/IWETH.sol';
+
+contract SafeBoxETH is Governable, ERC20, ReentrancyGuard {
+  using SafeMath for uint;
+  using SafeERC20 for IERC20;
+
+  ICErc20 public immutable cToken;
+  IWETH public immutable weth;
+
+  address public relayer;
+  bytes32 public root;
+  mapping(address => uint) public claimed;
+
+  constructor(
+    IWETH _weth,
```

Comment:
Do we need this argument? `_cToken.underlying()` is already supposed to be WETH?